metashare/api/models.py

```diff
@@ -289,6 +294,29 @@ def save(self, *args, **kwargs):
 
         return ret
 
+    def refresh_access_token(self):
+        """
+        This refreshes the access token in-band, so be careful.
+        """
+        self.config = refresh_access_token(
+            config=self.config, org_name="dev", login_url=self.login_url
+        )
+        self.save()
+
+    def get_login_url(self):
+        self.refresh_access_token()
+        org_config = OrgConfig(self.config, "dev")
```

Comment:
Correct me if I'm wrong, but I think `refresh_access_token` returns the `OrgConfig`, and then we're storing off just part of it, and then right away reconstructing a new `OrgConfig` here. Perhaps we could simplify by combining the model `refresh_access_token` and `get_login_url` methods?